sqlite_fdw.h

```diff
@@ -43,6 +43,8 @@
 
 #define CR_NO_ERROR 0
 
+#define SQLITE_FDW_BIT_DATATYPE_BUF_SIZE sizeof(sqlite3_int64)* CHAR_BIT + 1
```

Comment:
Could you add a space before `*` char: `sizeof(sqlite3_int64) * CHAR_BIT + 1`

Reply:
Added, see https://github.com/pgspider/sqlite_fdw/pull/83/commits/77ba282682e6c158be8fd04a6b956e234554b7be

Reply:
Thank for fixing. Please help me to resolve this thread.